kube/service.go

```diff
@@ -108,14 +126,14 @@ func NewClusterIPService(role *model.Role, headless bool, public bool, settings
 		if settings.CreateHelmChart {
 			// Backwards compatibility: If .kube.external_ips doesn't exist,
 			// use .kube.external_ip instead (as the single address)
-			externalIP = strings.Replace(`{{
+			externalIP = strings.Join(strings.Fields(`{{
```

Comment:
This seems to have nothing to to with templatized ports.

Reply:
> This seems to have nothing to to with templatized ports.

It doesn't, which is why it is in a commit by itself, and the commit doesn't have the Trello card name prefix.

It is just a tiny code cleanup thing that seems small enough to not need a separate PR.

But we can split it out if you want. In which case we should probably move your cleanup of `helm/config.go` into a separate PR as well.

Reply:
Ok. Accepted.